packages/cli/src/modules/build/commands/package/start/startPackage.ts

```diff
@@ -17,9 +17,26 @@
 import { PackageRole } from '@backstage/cli-node';
 import { startBackend, startBackendPlugin } from './startBackend';
 import { startFrontend } from './startFrontend';
+import { statSync, readdirSync } from 'fs';
+import { resolve, join, parse } from 'path';
+
+export function resolveEntryPath(
+  entrypoint: string = 'dev',
+  targetDir: string,
+): string {
+  const { dir: entryDir, name: entryName } = parse(entrypoint);
+  const entryPath = join(entryDir, entryName);
+  const parentPath = resolve(targetDir, entryDir);
+  const isFile = readdirSync(parentPath).some(base => {
```

Comment:
Hm I think you could start with stat on the exact full path and if that's a directory, perform one set of logic and otherwise perform another set of logic 🤔 Right? Haha I just feel uneasy with readdir on what could be NOT a directory to begin with, even if it works in practice. Also if you flip the logic you don't need a parent dir variable at all, I think.

Reply:
I simplified the logic, please tell me what you think 🙂